<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColormapWidget</class>
 <widget class="QWidget" name="ColormapWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LabelWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Colormap and labels</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Active colormap</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="colormapComboBox"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="LabelChooser">
        <item>
         <layout class="QVBoxLayout" name="leftLabel">
          <item alignment="Qt::AlignHCenter|Qt::AlignTop">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Left</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="leftComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="insertPolicy">
             <enum>QComboBox::InsertAtBottom</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignTop">
           <widget class="QLabel" name="lblLeftLabelName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="rightLabel">
          <item alignment="Qt::AlignHCenter|Qt::AlignTop">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Right</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="rightComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignTop">
           <widget class="QLabel" name="lblRightLabelName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnSwapLabels">
        <property name="text">
         <string>Swap labels</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
